doc/conf.py

```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'Spyder'
-copyright = '2009–2018 The Spyder Doc Contributors'
+copyright = "<p>&copy; 2009-2018 The Spyder Doc Contributors <br> Icons by <a href='http://www.freepik.com' target='_blank' title='Freepik'>Freepik</a> from <a href='https://www.flaticon.com/' target='_blank' title='Flaticon'>Flaticon.com</a> | <a href='https://freehtml5.co/icon-free-website-template-using-bootstrap/' target='_blank'>Template</a> designed by <a href='http://freehtml5.co/' target='_blank'>FreeHTML5.co</a> <br>Hugo port by <a href='https://github.com/SteveLane/' target='_blank'>SteveLane</a> | Lektor port by <a href='https://github.com/dalthviz/' target='_blank'>Dalthviz</a></p>"
```

Comment:
I'll do a full review in a few hours, but this should be the text that's currently present in the current footer, perhaps with "; licensed [MIT](https://opensource.org/licenses/MIT) tacked on to "Spyder Doc Contributors."

![image](https://user-images.githubusercontent.com/17051931/42190723-c0233958-7e2c-11e8-9638-8aa9d1d14735.png)